src/app/pages/search-result/search-result.component.ts

```diff
@@ -1,24 +1,58 @@
-import { Component, OnInit } from '@angular/core';
-import { BasicCharacterResponse, CharacterResponseData } from '../../interfaces/character-response-data.interface';
-import { BasicCharacter, Character } from '../../interfaces/character.interface';
+import { Component, OnDestroy, OnInit } from '@angular/core';
+import { Router } from '@angular/router';
+import { Subject, takeUntil } from 'rxjs';
```

Comment:
- move to top

Reply:
Why should I move it?
I understand that the standar is to have:
1] angular stuff
2] external libraries
3] relative paths
...

Am I wrong?